components/select_team/select_team.jsx

```diff
@@ -222,9 +239,16 @@ export default class SelectTeam extends React.Component {
                             defaultMessage='Teams you can join: '
                         />
                     </h4>
-                    <div className='signup-team-all'>
-                        {joinableTeamContents}
-                    </div>
+                    <InfiniteScroll
+                        callBack={this.fetchMoreTeams}
+                        endOfData={endofTeamsData}
+                        endOfDataMessage='No more teams to display'
```

Comment:
I don't think we need an end of teams message here as we previously did not have it. @esethna do we want to show a message when user reaches to the bottom of the list?

Reply:
@sudheerDev waiting on @esethna 's response before implementing this feedback

Reply:
Agree I don't think we need a message here

Reply:
@esethna and @sudheerDev let me change that...

Reply:
@sudheerDev I have made the `endOfDataMessage` prop optional in the `InfiniteScroll` component and it defaults to an empty string. For the infinite loading of our teams, I don't pass in the `endOfDataMessage` prop and it's therefore not visible when the last teams have been loaded.